### PR TITLE
Added note about removing any existing serialized schema files when switching an entity to be deployed as content.

### PR DESCRIPTION
**Pull request by AndyButland**
Came up in a support issue and having followed the recommendation added here would have avoided the problem.

Can be reviewed and go live at any time.

// @ronaldbarendse

### CODE REVIEW

File: 11/umbraco-deploy/deploy-settings.md [hunk 1]
```diff
@@ -189,10 +189,14 @@ If for any reason there was a need to revert to the previous implementation, the
 
 In order for Deploy to handle Forms data as content, you'll to ensure the `TransferFormsAsContent` setting is set to `true`. To transfer Forms data as schema, i.e. via .uda files committed to source control, use a value of `false`.
 
+On changing this value from `false` to `true`, make sure to remove any `.uda` files for Forms entities that have already been serialized to disk. These will no longer be updated. By deleting them you avoid any risk of them being processed in future and inadvertently reverting a form to an earlier state.
```
Comment (jonat123) [inline 11/umbraco-deploy/deploy-settings.md:192]:
```suggestion
{% hint style="info" %}
On changing this value from `false` to `true`, make sure to remove any `.uda` files for Forms entities that have already been serialized to disk. These will no longer be updated. By deleting them you avoid any risk of them being processed in the future and inadvertently reverting a form to an earlier state.
{% endhint %}
```

**Review by jonat123 — CHANGES_REQUESTED** [review]
@AndyButland Looks good!

I've made a couple of suggestions to put the note into a tip-box.

Let me know what you think :-)

**Comment by AndyButland** [comment]
Changes look good to me, I've committed them.

**Review by jonat123 — APPROVED** [review]
@AndyButland perfect. 

Merging in the PR :)

**Pull request merged**